File: tests/expected/global2.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

final int code_0 = 0;
final int code_1 = 1;
final String code_a = "a";
final String code_b = "b";
final Set<String> l_b = new Set.from([code_a]);
final Map<String, int> l_c = {code_b: code_0};
main(List<String> argv) {
  assert(l_b.contains("a"));
  print(sprintf("%s", ["OK"]));
}

File: tests/expected/global2.kt
val code_0 = 0
val code_1 = 1
val code_a = "a"
val code_b = "b"
val l_b = setOf(code_a)
val l_c = hashMapOf(code_b to code_0)

fun main(argv: Array<String>) {
    assert("a" in l_b)
    println("OK")
}
